ComputeClient/Compute.Client.UnitTests/Server20/ServerAccessorTests.cs

```diff
@@ -10,55 +10,55 @@
 
```

Comment:
Real changes start on line 266 - new test to ensure XML payload is unmarshalled correctly. Other changes are as a result of formatting in Visual Studio 2017.